lib/redis_cluster/client.rb

```diff
@@ -4,10 +4,20 @@ module RedisCluster
 
   class Client
 
-    def initialize(startup_hosts, global_configs = {})
+    def initialize(startup_hosts, configs = {})
       @startup_hosts = startup_hosts
-      @pool = Pool.new(global_configs)
+
+      # Extract configuration options relevant to Redis Cluster.
+
+      # force_cluster defaults to true to match the client's behavior before
+      # the option existed
+      @force_cluster = configs.delete(:force_cluster) { |_key| true }
```

Comment:
I think that `force_cluster` and the behavior around an array/non-array `startup_hosts` option should be in the `README`, but after taking a look, I figured that I'd try my hand at a more complete revamp that includes any new options that I add in here.